### PR TITLE
feat(params): Add customisable committer params

### PR DESCRIPTION
**Pull request by centrumek**
Default Case:

Screenshot 2023-10-10 at 19 14 29

Screenshot 2023-10-10 at 19 15 00

Customised Case:

Screenshot 2023-10-10 at 18 49 37

Screenshot 2023-10-10 at 18 51 13

### CODE REVIEW

**Comment by centrumek** [comment]
Feature on behalf of https://github.com/dailydotdev/action-devcard/issues/367

**Review by omBratteng — CHANGES_REQUESTED** [review]
Don't commit the dist file, I'll build a new one when I release a new version.

File: src/index.ts [hunk 1]
```diff
@@ -54,7 +54,9 @@ const validateDevcardIdAsUUID = (devcard_id: string): boolean => {
 		const branch = core.getInput('commit_branch')
 		const message = core.getInput('commit_message')
 		const filename = core.getInput('commit_filename')
-		const dryrun = core.getBooleanInput('dryrun')
+		const email = core.getInput('committer_email')
+		const name = core.getInput('committer_name')
+		const dryrun = false
```
Comment (omBratteng) [inline src/index.ts:59]:
```suggestion
		const dryrun = false
```

Don't set dryrun to static false

**Review by omBratteng — CHANGES_REQUESTED** [review]

File: src/index.ts (hunk 1 above)
Comment (centrumek) [reply src/index.ts:59]: Ahh sorry, I've changed that when testing locally

**Comment by centrumek** [comment]
@omBratteng, I've applied changes

**Review by omBratteng — APPROVED** [review]

**Pull request merged**